src/go/k8s/config/crd/bases/cluster.redpanda.com_redpandas.yaml

```diff
@@ -61,7 +61,7 @@ spec:
                   timeout:
                     description: Timeout is the time to wait for any individual Kubernetes
                       operation (like Jobs for hooks) during the performance of a
-                      Helm action. Defaults to '10m0s'.
+                      Helm action. Defaults to '15m0s'.
```

Comment:
I would combine first commit with this as any changes to api/go description/comment could be automatically generated by `make manifests`